docs/design/storage-node-deployment-windows-home.md

```diff
@@ -0,0 +1,165 @@
+# Storage Node Automatic Updates and Installation for Windows
+
+## Overview
+
+As more Storage Node Operators join the network we must ensure these nodes have a mechanism to automatically keep their nodes software up to date.
+If a Storage Node Operator fails to keep their node up to date with the minimum version required by satellites they will no longer be selected for upload or download requests.
+
+Currently we are using Docker for automatic updates which means we require Docker to be installed.
+Docker is being used for many other things as well, so we need to cover these cases as well.
+
+## Goals
+
+To deploy the automatic updates we need to handle the following cases that docker handles. Docker is not supported on windows home so we will ensure an automatic update system is built into the Storage nodes. 
+We also need to ensure that it plays nicely with all the common anti-viruses and firewalls.
+
+Docker is being used for:
+
+* Automatic Updates (with watchtower)
+* Restarting on crash
+* Logging (kind of)
+
+## Services
+
+* Installer
+    * Must be run with admin privileges.
+    * Installs the automatic updater binary and error gui application. (msi)
+    * Sets up automatic updater as a windows Service with sufficient privileges.
+* Automatic Updater (binary)
+    * Downloads storage node binary, Sets up storage node, and runs the watchdog process.
+    * Doesn't start the storage node if storage node has not created a valid config.
+    * Send error reports to satellite.
+    * Writes update related errors to log file.
+    * Watchdog process
+        * Monitors storage node by periodically sending messages to pulse endpoint on storage node and waiting for responses.
+        * Restarts the storage node if a crash/unresponsiveness is detected.
+* Storage Node (binary)
+    * Shares drive with satellite network.
+    * Writes storage node operation related errors to log file
+    * Has pulse endpoint that can be hit for knowing if the storage node is alive and healthy. 
+* Error gui application
+    * Shows errors from log file
+    * Notifies user of service errors.
+    * Saves last reported error timestamp to a file for knowing if there are unread errors.
+
+### Starting Storage Node binary on OS Start-Up
+
+We need to ensure that updater binary starts on computer start-up,
+without logging into the system, and this updater binary launches the storage node This is achieved when installing via the msi.
```

Comment:
This has nothing to do with MSI. MSI is an installer and irrelevant to this goal.